tests/test_load_results/test_mteb_results.py

```diff
@@ -145,5 +149,148 @@ def test_revision_layer(json_path, results_path):
         ], f"Unexpected path structure for version {json_version}: '{relative_path}'"
 
 
+@pytest.mark.xfail(reason="PairClassification and Reranking fail that test")
+@pytest.mark.parametrize(
+    "task, dataset",
+    [
+        (
+            "BitextMining",
+            {"sentence1": ["test"] * 2, "sentence2": ["test"] * 2, "id": ["test"] * 2},
+        ),
+        (
+            "Classification",
+            {"text": ["test"] * 2, "label": [1, 0]},
+        ),  # classification needs at least 2 classes
+        ("Clustering", {"sentences": [["test"]] * 2, "labels": [[0], [1]]}),
+        (
+            "PairClassification",
+            {
+                "sentence1": [["test"]] * 2,
+                "sentence2": [["test"]] * 2,
+                "labels": [[1]] * 2,
+            },
+        ),
+        (
+            "Reranking",
+            {
+                "query": ["test"] * 2,
+                "positive": [["test"]] * 2,
+                "negative": [["test"]] * 2,
+            },
+        ),
+        (
+            "STS",
+            {"sentence1": ["test"] * 2, "sentence2": ["test"] * 2, "score": [1] * 2},
+        ),
+        (
+            "Summarization",
+            {
+                "text": ["text"],
+                "human_summaries": [["text"]],
+                "machine_summaries": [["text"]],
+                "relevance": [[0.1]],
+            },
+        ),
+    ],
+)
+def test_main_score_in_task_result(task, dataset):
```

Comment:
I would delete these test and instead move the logic to:

```
tests/test_mteb::test_mteb_task
```

```
def test_mteb_task(task: Union[str, AbsTask], model_name: str):
    """Test that a task can be fetched and run"""
    model = SentenceTransformer(model_name)
    eval = MTEB(tasks=[task])
    results = eval.run(model, output_folder="tests/results", overwrite_results=True)
    # add test of results here
```

Reply:
I wrote it this way to ensure the test runs quickly without requiring data downloads or complex models. However, I'm open to making the change to align with the suggested solution

Reply:
The goal here is to make tests that are easy to maintain going forward. Would def. be more than happy to create a list of mock datasets for all the dataset and create two versions of the test test_mteb_task one for integration and one using only the mock datasets.